src/oemof_tabular_plugins/wefe/facades/crops.py

```diff
@@ -0,0 +1,1176 @@
+                )  # 0.096 water uptake constant (SIMPLE)
+
+                if has_irrigation:
+                    water_deficit = df.loc[index, "et_a"] - (row["tp"] - row["runoff"])
+                    df.at[index, "irrigation"] = max(water_deficit, 0)
+
+                delta_swc = (
+                    row["tp"]
+                    - row["runoff"]
+                    - df.loc[index, "deep_drain"]
+                    - df.loc[index, "et_a"]
+                    + df.loc[index, "irrigation"]
+                ) / rzd
+
+                df.loc[index, "swc"] = swc_cache + delta_swc
+                swc_cache = df.loc[index, "swc"]
+
+        def drought(et_p, et_a, s_water, **kwargs):
+            """Drought stress effect on plant growth (SIMPLE) [-]"""
+            arid = 1 - min(et_p, et_a) / et_p
+            f_drought = 1 - s_water * arid
+            return f_drought
+
+        def faster_senescence(cum_temp, f_heat, f_drought, i50maxh, i50maxw, **kwargs):
+            """Heat and drought stress effect on solar interception due to faster canopy senescence (SIMPLE) [-]"""
+            if cum_temp < 1:
+                delta_i50b = 0
+            else:
+                delta_i50b = (i50maxh * (1 - f_heat) + i50maxw * (1 - f_drought)) / 24
+            return delta_i50b
+
+        def solar_interception(
+            cum_temp, delta_i50b, t_sum, i50a, i50b, f_solar_max, **kwargs
+        ):
+            """
+            Interception of incoming radiation by plant canopy
+            according to cumulative temperature experienced (SIMPLE) [-]
+            """
+            cum_temp_to_reach_f_solar_max = i50a - 100 * np.log(
+                1 / 999
+            )  # derived from f_solar = 0.999f_solar_max
+            if cum_temp < 1:
+                f_solar = 0
+            elif cum_temp < cum_temp_to_reach_f_solar_max:
+                i50b += delta_i50b
+                f_solar = f_solar_max / (1 + np.exp(-0.01 * (cum_temp - i50a)))
+            elif cum_temp < t_sum:
+                i50b += delta_i50b
+                f_solar = f_solar_max / (1 + np.exp(0.01 * (cum_temp - (t_sum - i50b))))
+            else:
+                f_solar = 0
+            return f_solar
+
+        # Get attributes for biomass modelling from kwargs
+        self.elevation = kwargs.pop("elevation")
+        self.crop_type = kwargs.pop("crop_type")
+        self.sowing_date = kwargs.pop("sowing_date")
+        self.harvest_date = kwargs.pop("harvest_date")
+        self.has_irrigation = kwargs.pop("has_irrigation")
+
+        # Create DataFrame out of input profiles (time series), time_profile will be set as DatetimeIndex
+        profiles_dict = {
+            key.replace("_profile", ""): value
+            for key, value in kwargs.items()
+            if key.endswith("profile")
+        }
+        time_index = profiles_dict.pop("time")
+        profiles_df = pd.DataFrame(data=profiles_dict, index=time_index)
```

Comment:
This is very specific on formatting of the sequences files and not described to the user

Reply:
Why not using the profiles which are provided by the attributes, like `self.t_dp_profile`

Reply:
I specified the attributes for the time series as "<blabla>_profile", this way I can easily filter the time series out of all the kwargs instead of directly specifying them. 
To the user it shouldnt matter, for the attribute "ghi_profile" they can insert "banana" in the csv and as long as its a valid time series, everything will work out just fine.

self.t_dp_profile would also need to be set using kwargs.pop("t_dp_profile"), it does not really make things easier in my opinion.